SECURITY.md

```diff
@@ -0,0 +1,25 @@
+Security
+========
+
+Safety and data security is of utmost priority for the Mattermost community. If you are a security researcher and have discovered a security vulnerability in our code base, we appreciate your help in disclosing it to us in a responsible manner.
+
+Reporting security issues
+-------------------------
+
+**Please do not use GitHub issues for security-sensitive communication.**
+
+Security issues in the community test server, any of the open source codebases maintained by Mattermost, or any of our commercial offerings should be reported via email to [responsibledisclosure@mattermost.com](mailto:responsibledisclosure@mattermost.com). Mattermost is committed to working together with researchers and keeping them updated throughout the patching process. Researchers who responsibly report valid security issues will be publicly credited for their efforts&mdash;if they so choose.
+
+For a more detailed description of the disclosure process and a list of researchers who have previously contributed to the disclosure program, see [Report a Security Vulnerability](https://mattermost.com/security-vulnerability-report/) on the Mattermost website.
+
+Security updates
+----------------
+
+Mattermost has a mandatory upgrade policy, and updates are only provided for the latest 3 releases and Extended Support Release (ESR). Critical updates are delivered as dot releases. Details on security updates are announced 30 days after the availability of the update.
```

Comment:
```suggestion
Mattermost has a mandatory upgrade policy, and updates are only provided for the latest 3 releases and the current Extended Support Release (ESR). Critical updates are delivered as dot releases. Details of security patched security vulnerabilities are announced 30 days after the availability of the update fixing them.
```

Suggesting a clarification, although this might change the meaning/emphasis in a way you don't want?

Reply:
I'd go for the original; not everything included in a security update is necessarily a vulnerability, strictly speaking.

Reply:
```suggestion
Mattermost has a mandatory upgrade policy, and updates are only provided for the latest 3 releases and the currentExtended Support Release (ESR). Critical updates are delivered as dot releases. Details on security updates are announced 30 days after the availability of the update.
```

Makes sense, how about this bit though? Could just even be:

```suggestion
Mattermost has a mandatory upgrade policy, and updates are only provided for the latest 3 releases and the Extended Support Release (ESR). Critical updates are delivered as dot releases. Details on security updates are announced 30 days after the availability of the update.
```

if you don't like `current`.